devfiles/nodejs-basic/devfile.yaml

```diff
@@ -1,30 +1,55 @@
-apiVersion: 1.0.0
+schemaVersion: 2.0.0
 metadata:
-  generateName: nodejs-
+  name: nodejs
 projects:
   - name: express
-    source:
-      type: git
+    git:
       location: https://github.com/neeraj-laad/nodejs-basic-starter.git
 components:
-  - type: dockerimage
-    alias: node12
-    image: node:12
-    memoryLimit: 1024Mi
-    mountSources: true
-    endpoints:
-      - name: nodejs
-        port: 3000
+  - container:
+      image: registry.access.redhat.com/ubi8/nodejs-12:1-36
+      mountSources: true
+      name: node12
+      memoryLimit: 1024Mi
+      endpoints:
+        - name: nodejs
+          targetPort: 3000
+          configuration:
+            protocol: tcp
+            scheme: http
+            type: terminal
+        - name: debug
+          targetPort: 9229
+          configuration:
+            protocol: tcp
+            scheme: http
+            type: terminal
 commands:
-  - name: devBuild
-    actions:
-      - type: exec
-        component: node12
-        command: npm install
-        workdir: ${CHE_PROJECTS_ROOT}/express
-  - name: devRun
-    actions:
-      - type: exec
-        component: node12
-        command: npm start
-        workdir: ${CHE_PROJECTS_ROOT}/express
+  - exec:
+      id: download app dependencies
+      commandLine: "npm install"
+      component: node12
+      workingDir: ${CHE_PROJECTS_ROOT}/express
+      group:
+        kind: build
+  - exec:
+      id: run the app
+      commandLine: "npm start"
+      component: node12
+      workingDir: ${CHE_PROJECTS_ROOT}/express
+      group:
+        kind: run
+  # - exec:
```

Comment:
I'd suggest deleting rather than commenting out, git keeps the history, it doesn't have to be kept in-file.